libraries/opensk/src/ctap/mod.rs

```diff
@@ -3396,22 +3360,6 @@ mod test {
         assert!(matches!(response, Ok(_)));
     }
 
-    #[test]
-    fn test_check_user_presence_timeout() {
-        // This will always return timeout.
-        fn user_presence_timeout() -> UserPresenceResult {
-            Err(UserPresenceError::Timeout)
-        }
-
-        let mut env = TestEnv::default();
-        env.user_presence().set(user_presence_timeout);
-        let response = check_user_presence(&mut env, DUMMY_CHANNEL);
-        assert!(matches!(
-            response,
-            Err(Ctap2StatusCode::CTAP2_ERR_USER_ACTION_TIMEOUT)
```

Comment:
This is the removed test.

Reply:
We need to change `TestClock` to be thread-safe with interior mutability (so with `now_ms: Arc<Mutex<usize>>` for example). And then we can control the `Arc<Mutex<usize>>` from the test function.

Reply:
Thanks for the pointer, fixed.